src/astar/astar_process.cpp

```
@@ -0,0 +1,102 @@
+Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+
+ ********************************************************************PGR-GNU*/
+
+#include "process/astar_process.h"
+
+extern "C" {
+#include "c_common/postgres_connection.h"
+#include "c_common/time_msg.h"
+}
+
+#include <string>
```

Cppcheck (2.19.0)

[information] Include file

(missingIncludeSystem)